util.go

```diff
@@ -77,7 +77,7 @@ func absPathify(inPath string) string {
 
 // Check if File / Directory Exists
 func exists(path string) (bool, error) {
-	_, err := v.fs.Stat(path)
+	_, err := os.Stat(path)
```

Comment:
@bep not a bad merge, my overzealous goimports tool got out of hand.  Fixing it.  Good catch.